rest/route/middleware.go

```diff
@@ -322,13 +330,21 @@ func RequiresDistroPermission(permission string, level evergreen.PermissionLevel
 	if !evergreen.AclCheckingIsEnabled {
 		return &noopMiddleware{}
 	}
+	defaultRoles, err := evergreen.GetEnvironment().RoleManager().GetRoles(evergreen.UnauthedUserRoles)
+	if err != nil {
+		grip.Critical(message.WrapError(err, message.Fields{
+			"message": "unable to get default roles",
+		}))
+		return nil
```

Comment:
Is it okay to return nil here? Also below this.

Reply:
I made it continue on error here, so the end result is that we won't give any security to people who aren't logged in